src/net/sourceforge/kolmafia/request/RelayRequest.java

```diff
@@ -923,6 +923,31 @@ public boolean sendBreakPrismWarning(final String urlString) {
       return true;
     }
 
+    // In Fall of the Dinosaurs, you will lose access to the Dinostaur and can therefore not spend
+    // your dinodollars
+
+    if (KoLCharacter.inDinocore()) {
```

Comment:
How about some tests in RelayRequestWarningsRequestTest?
RelayRequest.sendBreakPrismWarning has a warning for leaving Robocore for a similar reason.
No tests for Kingdom of Exploathing and Plumber paths, alas. for 

That whole file could used some @Nested classes - and Cleanups.